hackathon/react-native-heroes/components/Hero.tsx

```diff
@@ -0,0 +1,165 @@
+import {
+  View,
+  Text,
+  Image,
+  StyleSheet,
+  TouchableOpacity,
+  Linking,
+  Modal,
+  Pressable,
+  TouchableWithoutFeedback,
+} from "react-native";
+
+import React, { useState } from "react";
+
+type Props = {
+  avatar: string;
+  name: string;
+  urlGithub: string;
+  urlTwitter: string;
+};
+
+const Hero = (hero: Props) => {
+  const [modalVisible, setModalVisible] = useState(false);
+  const avatar = hero.avatar;
+  return (
+    <View style={styles.container}>
+      <Modal
+        animationType="slide"
+        transparent={true}
+        visible={modalVisible}
+        onRequestClose={() => {
+          setModalVisible(!modalVisible);
+        }}
+      >
+        <View style={styles.centeredView}>
+          <View style={styles.modalView}>
+            <Text style={styles.modalText}>Nom du développeur</Text>
+            <Image
+              source={{ uri: avatar }}
+              style={{
+                width: 200,
+                height: 200,
+                borderRadius: 10,
+                borderWidth: 1,
+                alignContent: "center",
+                justifyContent: "center",
+                alignItems: "center",
+              }}
+            />
+            <Text>Description du développeur</Text>
+            <Pressable
+              style={styles.buttonModal}
+              onPress={() => setModalVisible(!modalVisible)}
+            >
+              <Text>Hide Modal</Text>
+            </Pressable>
+          </View>
+        </View>
+      </Modal>
+      <TouchableWithoutFeedback onPress={() => setModalVisible(!modalVisible)}>
```

Comment:
**Do not use unless you have a very good reason.** 
https://reactnative.dev/docs/touchablewithoutfeedback

`<TouchableOpacity>` is better